script/release

```diff
@@ -13,8 +13,7 @@ echo "FormRenderer.VERSION = '$version'" > src/version.coffee
 grunt all
 grunt test
 git commit -am "release $version"
-git tag $version
-git push origin master --tags
+git push origin repeating-sections --tags
```

Comment:
whats up with `repeating-sections` being hardcoded here? sounds like we should make the branch name an optional second arg

Reply:
Oh, I didn't realize I checked in that change. I'll remove it -- I had just changed it because I didn't want to merge into master yet, but I did want to release. Thanks for catching this (but good call on making it an optional arg).